app.go

```diff
@@ -42,6 +42,7 @@ func New(opts ...Option) *App {
 		sigs:             []os.Signal{syscall.SIGTERM, syscall.SIGQUIT, syscall.SIGINT},
 		registrarTimeout: 10 * time.Second,
 		stopTimeout:      10 * time.Second,
+		afterStopTimeout: 5 * time.Second,
```

Comment:
The current code uses a canceled context whose `Done()` will return a closed channel behaving the same as a timed out context. I think it defaults to timeout immediately before modification rather than never timeout.

Reply:
I hope it can remain the same as before, and if you need to control the time, you can set the timeout yourself.

Reply:
Sorry for the late reply. I have updated the PR to set default afterstop timeout to zero. Kratos will pass a canceled context to AfterStop functions as the same as before if user don't specify the AfterStop timeout.